src/StudyViewer/StudyCard.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Card, Collapse } from 'antd';
+import { PlusCircleOutlined, MinusCircleOutlined } from '@ant-design/icons';
+import { ReduxStudyDetails } from './reduxer';
+import './StudyViewer.css';
+
+const { Panel } = Collapse;
+
+class StudyCard extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      panelExpanded: props.initialPanelExpandStatus,
+    };
+  }
+
+    onCollapseChange = () => {
+      this.setState(prevState => ({
+        panelExpanded: !prevState.panelExpanded,
+      }));
+    };
+
+    render() {
+      return (
+        <Card
+          className='study-viewer__card'
+          title={this.props.data.title}
+        >
+          <Collapse
+            defaultActiveKey={(this.state.panelExpanded) ? ['1'] : []}
+            expandIcon={({ isActive }) =>
+              ((isActive) ? <MinusCircleOutlined /> : <PlusCircleOutlined />)}
+            onChange={this.onCollapseChange}
+            ghost
+          >
+            <Panel
+              header={(this.state.panelExpanded) ? 'Hide details' : 'Show details'}
+              key='1'
+            >
+              <ReduxStudyDetails
+                data={this.props.data}
+                fileData={this.props.fileData}
+                displayLearnMoreBtn
```

Comment:
i see in your dev env that it works but i can't find where `displayLearnMoreBtn` is toggled on/off 😅

Reply:
the default value of this `ReduxStudyDetails` props in `StudyDetails` is `false`, in here is just passing the `true` value over